docs/example.md

```
@@ -0,0 +1,190 @@
+
+_This text will be italic_  
+**This text will be bold**  
+**_This text will be both italic and bold_**
+
+### 3. Lists
+
+#### Unordered List
+
+- Item 1
+- Item 2
+    - Subitem 2.1
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)